openapiart/openapiartprotobuf.py

```diff
@@ -183,14 +186,15 @@ def _get_description(self, openapi_object):
             return "Description missing in models"
 
     def _write_enum_msg(self, enum_msg_name, enums):
-        enum_msg = enum_msg_name.split(".")
-        self._write("message {} {{".format(enum_msg[0]), indent=1)
-        self._write("enum Enum {", indent=2)
+        """Follow google developers style guide for enums
+        - reference: https://developers.google.com/protocol-buffers/docs/style#enums
+        """
+        self._write("enum {} {{".format(enum_msg_name.replace(".", "")), indent=1)
```

Comment:
I think we'll need to go back to following inclusion of UNSPECIFIED.
Kudos for fixing names to use proper camel case.

```proto
message Choice {
    enum Enum {
        UNSPECIFIED = 0;
        LACP = 1;
        STATIC = 2;
    }
}

  optional Choice.Enum choice = 1 [
    (fld_meta).default = "Choice.Enum.LACP",
    (fld_meta).description = "The type of LAG protocol."
  ];

```

Otherwise, enum scoping rules will mess things up in cases where there are two enum definitions in a given message.

I confirmed the same and was getting the following message (when running against models/artifacts/openapi.yaml)

```
otg.proto:740:5: "UNSPECIFIED" is already defined in "otg.DeviceBgpv4".
```

because DeviceBgpv4 has two enums defined inside it, 
- AsType
- AsNumberWidth